_posts/2023-11-02-release.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Release 2023.44
+author: Priya
+date: 2023-11-02
+category: Jekyll
+layout: post
+---
+### Enhancements:
+
+#### Date Difference Calculation - Now with System Time and Specific Date Options:
+Calculate date differences more effectively with additional choices. You can choose between system time, specific dates, and even set calculations to the current date-time in UTC. The date difference component is also conveniently displayed in pipelines (Years, Months, Weeks, etc).
+
+#### Insert Current Date and Time - Now with UTC Option:
+Creating a column and inserting the current date and time is easier than ever. You can now choose to insert the current date and time in UTC using the label and insert value rule, giving you more control over your data.
+
+#### Improved Folder Deletion Handling for Dependency Issues:
+We've enhanced the folder deletion process to ensure consistent behavior with deletion operations. Now, items inside the folder are deleted first, and only after that, the folder is removed. This prevents issues with dependencies during dataset (ds) deletion.
+
+#### Improved User Experience During Instance Wake-Up:
+Enhancing the user experience when instances are waking up after periods of inactivity. Instead of encountering sluggishness upon login, users will now see a informative blocking modal message.This message will alert users that their account was snoozed due to inactivity and inform them that it's currently powering up.
+
+### Defect Fixes:
+* Fixed an issue where multiple DS or folder deletions in progress caused app to nearly stop working.
+* Resolved the problem of being unable to create an SFTP identity via an SSH key.
+* The Metric issue has been fixed, and the condition description now shows the current time when the metric is updated.
+* Data library now correctly uploads all files inside when multiple files are added to a folder.
+* Deleted projects no longer continue to appear in the sidebar.
+* Fixed the continuous loader issue in SSH Key-based connections while generating an SSH key.
```

Comment:
We can remove this. This is kind of captured in the previous statement about ssh key